src/patternfly/utilities/variables.scss

```diff
@@ -23,22 +23,21 @@ $pf-global--space-size-root: 16 !default; // for spaces
 // Colors
 
 // Background color
-$pf-global--BackgroundColor--light:      $pf-color-white !default;
-$pf-global--BackgroundColor--light-alt:  $pf-color-black-200 !default;
-// ADD one more light background
-$pf-global--BackgroundColor--light-alt:  $pf-color-black-200 !default;
+$pf-global--BackgroundColor--light:       $pf-color-black-200 !default;
+$pf-global--BackgroundColor--lighter:     $pf-color-black-100 !default;
+$pf-global--BackgroundColor--lightest:    $pf-color-white !default;
```

Comment:
Be careful with names like "light, lighter, lightest" as it leaves no values in between, might we ever need those.

Reply:
thanks @thomasmaas. I agree, this is a discussion I want to have with the designers to get a better understanding of direction for the future. Do you have any thoughts on what we could use for naming here?